wagtail_localize/test/migrations/0003_nontranslatablechildobject_testgeneratetranslatablefieldspage_translatablechildobject.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 3.0.8 on 2020-08-21 16:52
```

Comment:
Greatest migration filename of all time.

Reply:
Oh, yeah...